server/src/main/java/org/candlepin/resource/ConsumerTypeResource.java

```diff
@@ -151,7 +151,7 @@ public ConsumerTypeDTO create(
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
     public ConsumerTypeDTO update(
-        @ApiParam(name = "consumerType", required = true) ConsumerTypeDTO dto) throws BadRequestException {
+        @ApiParam(name = "type", required = true) ConsumerTypeDTO dto) throws BadRequestException {
```

Comment:
Why change the name of the API's property on the swagger output? Accident due to mass refactor/rename with IDE? Same with the other two changes in this file.